RPerformanceTracking/Private/_RPTTrackingManager.m

```diff
@@ -122,9 +122,10 @@ - (instancetype)init
             [self addEndMetricObservers];
             [_sender start];
             
-            // Profile main thread to check if it is running for > 0.4 s
-            _watcher = _RPTMainThreadWatcher.new;
+            // Profile main thread to check if it is running for > threshold time
+            _watcher = [_RPTMainThreadWatcher.alloc initWithThreshold:0.4];
```

Comment:
We can have a static const NSTimeInterval BLOCK_THRESHOLD = 0.4;
and _watcher = [_RPTMainThreadWatcher.alloc initWithThreshold:BLOCK_THRESHOLD];